examples/image_classification/README.md

```diff
@@ -0,0 +1,393 @@
+majorityVoteParameters.setHistorySize(10);
+QualityControlConfig majorityVoteControlConfig = new QualityControlConfig(
+        new CollectorConfig.MajorityVote(majorityVoteParameters),
+        List.of(majorityVoteRuleConfig)
+);
+```
+
+```java
+RuleConfig goldenSetRuleConfig = new RuleConfig(
+        List.of(
+                new RuleCondition.GoldenSetCorrectAnswersRate(CompareOperator.LT, 60.0),
+                new RuleCondition.GoldenSetAnswersCount(CompareOperator.GTE, 3)
+        ),
+        new RuleAction.RestrictionV2(
+                new RuleAction.RestrictionV2.Parameters(
+                        UserRestrictionScope.PROJECT,
+                        10, DurationUnit.DAY,
+                        "Golden set")
+        ));
+
+QualityControlConfig goldenSetControlConfig = new QualityControlConfig(
+        new CollectorConfig.GoldenSet(),
+        List.of(goldenSetRuleConfig)
+);
+```
+
+```java
+QualityControl qualityControl = new QualityControl(List.of(
+        incomeSumControlConfig,
+        skippedInRowControlConfig,
+        majorityVoteControlConfig,
+        goldenSetControlConfig
+));
+pool.setQualityControl(qualityControl);
+```
+
+Specify the number of tasks per page. For example: 9 main tasks and 1 control task.
+
+```java
+pool.setMixerConfig(new MixerConfig(
+                9, // real tasks count
+                1, // golden tasks count
+                0  // training tasks count
+                ));
+```
+
+Create pool.
+
+```java
+ModificationResult<Pool> poolCreationResult = clientFactory.getPoolClient().createPool(pool);
+Pool createdPool = poolCreationResult.getResult();
+String poolId = createdPool.getId();
+```
+
+Link to open in web interface: `https://toloka.yandex.com/requester/project/<project_id>/pool/<pool_id>`
+
+## Preparing and uploading tasks
+
+This example uses a small data set with images.
+
+The dataset used is collected by Toloka team and distributed under a [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/).
+
+Dataset looks like:
+
+![dataset preview](img/dataset_preview.png)
+
+Upload dataset into file `dataset.tsv`.
+```bash
+curl https://tlk.s3.yandex.net/dataset/cats_vs_dogs/toy_dataset.tsv --output dataset.tsv
+```
+
+Divide the dataset into two. One for tasks and one for [Control tasks](https://yandex.com/support/toloka-requester/concepts/task_markup.html).
+
+**Note**. Control tasks are tasks with the correct response known in advance. 
+They are used to track the performer's quality of responses. The performer's response is compared to the response you provided. If they match, it means the performer answered correctly.
+
+```java
+var dataset = readTSV("src/main/java/examples/dataset.tsv");
+var goldenDataset = dataset.subList(0, 15);
+var taskDataset = dataset.subList(15, dataset.size());
+```
+
+Function `readTSV` takes file name and returns `List<Map<String, String>>`, where each map keys are colunm names in file 
```

Comment:
please attach method `readTSV` herer

Reply:
Variant of `readTSV` implementation is in file `ImageClassification.java`. Add information about it in `README.md`